homeassistant/components/nightscout/config_flow.py

```diff
@@ -16,7 +16,7 @@
 
 _LOGGER = logging.getLogger(__name__)
 
-DATA_SCHEMA = vol.Schema({vol.Required(CONF_URL): str, vol.Optional(CONF_API_KEY): str})
+DATA_SCHEMA = vol.Schema({vol.Required(CONF_URL): str, vol.Optional(CONF_API_KEY): str, vol.Required('Units'): vol.In(["mmol/l", "mg/dl"])})
```

Comment:
Let's create a minor version config entry migration where we set the old one as default

Reply:
Also use a constant for `Units` and make it snakecase

Reply:
I added the mg/dL as default and moved to a constant